duo_wordpress.php

```diff
@@ -156,6 +156,11 @@ function duo_auth_enabled(){
             return false; //allows the XML-RPC protocol for remote publishing
         }
 
+        if (defined('REST_REQUEST') && REST_REQUEST && duo_get_option('duo_rest', '') == 'off') {
```

Comment:
If I'm understanding correctly, the default here is that if the user has not set an option, we will default to NOT allowing REST requests prior to the option being toggled.